.travis.yml

```diff
@@ -1,27 +1,57 @@
-sudo: false
 language: python
 
 env:
-    - CONDA="python=2.7"
-    - CONDA="python=3.3"
-    - CONDA="python=3.4"
+  - CONDA_PYTHON_VERSION="2"
+  - CONDA_PYTHON_VERSION="3"
```

Comment:
First, I am not a Travis configuration expert. That said, the original configuration file did not seem to be testing all the Python versions. I followed instructions and recommendations made here: http://conda.pydata.org/docs/travis.html. It is definitely boilerplate, but I am not sure if it is outdated. I did like the recommendation of using ``conda build`` inside the test script to catch any issues with the build scripts.